server/src/uuid/model/entity/mod.rs

```diff
@@ -990,6 +994,68 @@ impl Entity {
     }
 }
 
+impl Entity {
+    pub async fn deleted_entities<'a, E>(
+        payload: &deleted_entities_query::Payload,
+        executor: E,
+    ) -> Result<Vec<deleted_entities_query::DeletedEntity>, operation::Error>
+    where
+        E: Executor<'a>,
+    {
+        let date_database: Option<DateTime> = match payload.after.as_ref() {
+            Some(date) => {
+                let date_chrono = match chrono::DateTime::parse_from_rfc3339(date) {
+                    Ok(parsed_date) => parsed_date.with_timezone(&Utc),
+                    Err(_) => {
+                        return Err(operation::Error::BadRequest {
+                            reason: "The date format should be YYYY-MM-DDThh:mm:ss{Timezone}"
+                                .to_string(),
+                        })
+                    }
+                };
+                Some(DateTime::from(date_chrono))
+            }
+            None => None,
+        };
+
+        let mut deleted_entities: Vec<deleted_entities_query::DeletedEntity> = Vec::new();
+        let result = sqlx::query!(
+            r#"
+                select uuid_id, max(event_log.date) as date
+                from event_log, uuid, instance, entity
+                where uuid.id = event_log.uuid_id
+                    and (? is null or event_log.date > ?)
+                    and (? is null or instance.subdomain = ?)
+                    and instance.id = entity.instance_id
+                    and entity.id = event_log.uuid_id
+                    and event_log.event_id = 10
+                    and uuid.trashed = 1
+                    and uuid.discriminator = 'entity'
+                group by uuid_id
+                order by date
+                limit ?
+            "#,
+            date_database,
+            date_database,
+            payload.instance,
+            payload.instance,
+            payload.first,
+        )
+        .fetch_all(executor)
+        .await?;
+
+        for entity in result {
+            let deletion_date: DateTime = entity.date.unwrap().into();
```

Comment:
I've tested and it works good. We are though uneasy with the use of `unwrap` here, since it could put the app in an unrecoverable state. See v.g. [this discussion](https://stackoverflow.com/questions/39477684/should-i-avoid-unwrap-in-production-application).